modules/kelbo.py

```diff
@@ -0,0 +1,30 @@
+from .base import Module
+import random
+import re
+
+class Kelbo(Module):
+    DESCRIPTION = '____ __ _____'
+    ARGC = 1
```

Comment:
If you specify `ARGC = 1`, it will fail and throw the user an error when it doesn't have that many arguments, so I'd say 0 so your random option will work